containers/buildinfo.go

```diff
@@ -40,6 +47,8 @@ func GetBuildInfo(ctx context.Context, d *dagger.Client, dir *dagger.Directory,
 		WithMountedDirectory("/src", dir).
 		WithWorkdir("/src")
 
+	enterpriseSha, _ := enterpriseCommit(ctx, container)
```

Comment:
This is the only use of this func, if we don't care about the `err` we could remove it from the func.

Reply:
Yeah good call.